MAINTAINERS.md

```diff
@@ -13,24 +13,24 @@ for general contribution guidelines.
 - [arjan-bal](https://github.com/arjan-bal), Google LLC
 - [arvindbr8](https://github.com/arvindbr8), Google LLC
 - [atollena](https://github.com/atollena), Datadog, Inc.
-- [cesarghali](https://github.com/cesarghali), Google LLC
 - [dfawley](https://github.com/dfawley), Google LLC
 - [easwars](https://github.com/easwars), Google LLC
 - [erm-g](https://github.com/erm-g), Google LLC
 - [gtcooke94](https://github.com/gtcooke94), Google LLC
-- [menghanl](https://github.com/menghanl), Google LLC
 - [purnesh42h](https://github.com/purnesh42h), Google LLC
-- [srini100](https://github.com/srini100), Google LLC
 - [zasweq](https://github.com/zasweq), Google LLC
 
 ## Emeritus Maintainers (in alphabetical order)
-- [adelez](https://github.com/adelez), Google LLC
-- [canguler](https://github.com/canguler), Google LLC
-- [iamqizhao](https://github.com/iamqizhao), Google LLC
-- [jadekler](https://github.com/jadekler), Google LLC
-- [jtattermusch](https://github.com/jtattermusch), Google LLC
-- [lyuxuan](https://github.com/lyuxuan), Google LLC
-- [makmukhi](https://github.com/makmukhi), Google LLC
-- [matt-kwong](https://github.com/matt-kwong), Google LLC
-- [nicolasnoble](https://github.com/nicolasnoble), Google LLC
-- [yongni](https://github.com/yongni), Google LLC
+- [adelez](https://github.com/adelez),
```

Comment:
formatting: Should we get rid of the trailing `,`?

Reply:
Yes, if we don't know their company we should either write "Unknown" or remove the comma (not sure which is better).  Or, leave Google LLC in there as the company they were at while they were a gRPC-Go maintainer?

Reply:
Removed comma. Removing org for all emeritus for symmetry. Difficult to track once someone moves out of active maintainer list.